x-pack/plugins/integration_assistant/common/api/categorization/categorization_route.schema.yaml

```diff
@@ -19,15 +19,15 @@ paths:
               type: object
               required:
                 - packageName
-                - datastreamName
+                - dataStreamName
```

Comment:
The casing for the "data stream" word was inconsistent, I just searched & replaced